### PR TITLE
[console] Add new plugin

### PR DESCRIPTION
**Pull request by k-hagio**
Add a new plugin for collecting information about console and keyboard
including the status of console and Scroll Lock flag.

Signed-off-by: Kazuhito Hagio <k-hagio-ab@nec.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] If this commit closes an existing issue, is the line `Closes: #ISSUENUMBER` included in an independent line?
- [x] If this commit resolves an existing pull request, is the line `Resolves: #PRNUMBER` included in an independent line?

### CODE REVIEW

File: sos/plugins/console.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    profiles = ('system',)
+    packages = ('kbd',)
+
+    def setup(self):
+        self.add_copy_spec("/proc/consoles")
+
+        self.add_cmd_output("fgconsole")
+
+        for tty in glob("/dev/tty[0-8]"):
+            self.add_cmd_output("kbdinfo -C " + tty + " gkbled")
```
Comment (TurboTurtle) [inline sos/plugins/console.py]: We could reduce this to a single `add_copy_spec()` call via:

```python
self.add_copy_spec([
    "kbdinfo -C %s gkbled" % tty for tty in glob.glob("/dev/tty[0-8]")
])

which imo also improves readability.

File: sos/plugins/console.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from glob import glob
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Console(Plugin, RedHatPlugin):
```
Comment (TurboTurtle) [inline sos/plugins/console.py]: @BryanQuigley @battlemidget any desire to have this in Ubuntu as well?

File: sos/plugins/console.py (hunk 1 above)
Comment (k-hagio) [reply sos/plugins/console.py]: Thanks, will update so.

File: sos/plugins/console.py (hunk 2 above)
Comment (BryanQuigley) [reply sos/plugins/console.py]: Yes, please -- although I'll still +1 if it's not added.

Comment (k-hagio) [reply sos/plugins/console.py]: I'll add Ubuntu. Thanks.

**Pull request closed without merging**